src/Bicep.MSBuild.E2eTests/examples/fail/fail.proj

```diff
@@ -0,0 +1,20 @@
+<!--
+  Do not include this project in the solution. It is intended to validate our MSBuild task.
+-->
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+  </PropertyGroup>
+
+  <!--
+    Pickup latest available packages (including prerelease) from local feed configured in NuGet.config.
+  -->
+  <ItemGroup>
+    <PackageReference Include="Azure.Bicep.CommandLine.$(RuntimeSuffix)" Version="*-*" />
```

Comment:
>*-* [](start = 82, length = 3)

This is not ok to use in a normal project, but for test purposes it's a great way of picking up the latest package from the local feed.